state/src/applicator/wallet_index.rs

```diff
@@ -147,6 +147,15 @@ impl StateApplicator {
                     .get_order_metadata(wallet_id, id)?
                     .ok_or(StateApplicatorError::MissingEntry(ERR_NO_METADATA))?;
 
+                // TODO: Here is where we should emit cancellation metrics.
```

Comment:
this comment is relevant to a design that is not settled decided on, we should remove it because it informs nothing about the code.

In particular, i disagree that we'd want to fetch prices here. that would make sense for a metrics collector or a service external to the relayer to do. the metrics export interface should be dead simple and almost always one liners.

but aside, let's remove this comment til we decide what we'll do

Reply:
whoops, this is a completely outdated comment (before I thought of the worker idea), will remove